event.json

```diff
@@ -0,0 +1,14 @@
+{
+  "pull_request": {
+    "number": 1,
+    "title": "Test PR",
+    "body": "This is a test pull request"
```

Comment:
The body of the pull request is too vague. It should provide more details about the changes made or the purpose of the pull request.